### PR TITLE
7358 step1 visual

### PR DESCRIPTION
**Pull request by jordiwes**
*Issue #:* /bcgov/entity#7358

*Description of changes:*
- validation style changes and behavior
- button style changes
- re-do fee ui
- change add to a button with drop down

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

**Review by doug-lovett — APPROVED** [review]
Quick scan, looks good.

File: ppr-ui/src/components/common/RegistrationFee.vue [hunk 1]
```diff
@@ -119,27 +106,75 @@ export default defineComponent({
 })
 </script>
 
-<style lang="scss" module>
+<style lang="scss" scoped>
```
Comment (kialj876) [inline ppr-ui/src/components/common/RegistrationFee.vue]: Why the change back to scoped?

File: ppr-ui/src/components/common/Stepper.vue [hunk 2]
```diff
@@ -68,7 +78,7 @@ export default class Stepper extends Vue {
   display: flex;
   justify-content: space-evenly;
   margin: 0;
-  padding: 2rem 0;
+  padding: 2rem 0 0 0;
```
Comment (kialj876) [inline ppr-ui/src/components/common/Stepper.vue:82]: I know you're just changing existing stuff, but we usually try to keep the padding/margin stuff in the html if possible. Also this scoped styling slipped in, if you could alter it to module that would be great

File: ppr-ui/src/components/registration/RegistrationBar.vue [hunk 3]
```diff
@@ -38,39 +159,60 @@ export default defineComponent({
...
       ...toRefs(localState)
     }
   }
 })
 </script>
 
-<style lang="scss" module>
-@import '@/assets/styles/theme.scss';
-#registration-btn {
+<style lang="scss" scoped>
```
Comment (kialj876) [inline ppr-ui/src/components/registration/RegistrationBar.vue]: scoped again?

File: ppr-ui/src/views/ReviewConfirm.vue [hunk 4]
```diff
@@ -145,14 +149,20 @@ export default class ReviewConfirm extends Vue {
...
+  @Watch('registrationIncomplete')
+  private registrationIncomplete (): void {
+    this.showStepErrors = true
+    alert('Error saving draft. Replace when design complete.')
+  }
 }
 </script>
 
-<style lang="scss" module>
+<style lang="scss" scoped>
```
Comment (kialj876) [inline ppr-ui/src/views/ReviewConfirm.vue:165]: and again :)

File: ppr-ui/src/views/LengthTrust.vue [hunk 5]
```diff
@@ -107,7 +108,8 @@ export default class LengthTrust extends Vue {
 
   private get breadcrumbs (): Array<BreadcrumbIF> {
     const registrationBreadcrumb = tombstoneBreadcrumbRegistration
-    registrationBreadcrumb[2].text = this.getRegistrationType?.registrationTypeUI || 'New Registration'
+    registrationBreadcrumb[2].text =
+      this.getRegistrationType?.registrationTypeUI || 'New Registration'
```
Comment (kialj876) [inline ppr-ui/src/views/LengthTrust.vue:112]: I think you should add this new one to the bread crumb unit tests

**Review by kialj876 — COMMENTED** [review]
Looks very good, just the style going back to module is what we want I think, and a few small comments/requests I had

File: ppr-ui/src/views/LengthTrust.vue (hunk 5 above)
Comment (jordiwes) [reply ppr-ui/src/views/LengthTrust.vue:112]: I added a new tombstone test

File: ppr-ui/src/views/ReviewConfirm.vue (hunk 4 above)
Comment (jordiwes) [reply ppr-ui/src/views/ReviewConfirm.vue:165]: I changed them all to module

File: ppr-ui/src/components/registration/RegistrationBar.vue [hunk 6]
```diff
@@ -206,34 +207,13 @@ export default defineComponent({
...
-.registration-select {
-  background-color: $BCgovBlue3;
-  color: white;
-}
-::v-deep .v-list-item .v-list-item__title {
-  font-size: 0.875rem;
-  color: $gray9;
-}
-::v-deep .v-list-item:hover {
+:v-deep .v-list-item:hover {
```
Comment (kialj876) [inline ppr-ui/src/components/registration/RegistrationBar.vue]: this works?? Ah I wish my past self new this

**Review by kialj876 — APPROVED** [review]
Looks good, thanks for making those changes :)

File: ppr-ui/src/components/registration/RegistrationBar.vue (hunk 6 above)
Comment (jordiwes) [reply ppr-ui/src/components/registration/RegistrationBar.vue]: lol, it doesn't good catch. i popped in one more change.

Comment (kialj876) [reply ppr-ui/src/components/registration/RegistrationBar.vue]: oh okay haha

**Review by kialj876 — APPROVED** [review]
